src/components/modals/AddUserModal.js

```diff
@@ -91,16 +92,18 @@ export const AddUserModal = hh(class AddUserModal extends Component {
       roles: this.state.updatedRoles
     };
     switch (this.state.mode) {
-      case 'Add':
+      case 'Add': {
```

Comment:
New linting rule in place that enforces block scoping of variables within switch cases. Without them every variable defined in each case (let, var, const) will be hoisted to the top of the function. The ideal is to initialize them when the case is in effect, so using braces to scope them is encouraged.